### PR TITLE
ft: S3C-3190 bucket notification populator

### PR DESCRIPTION
**Pull request by naren-rajendran**
New populator for bucket notification extension.

### CODE REVIEW

File: bin/notification.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const werelogs = require('werelogs');
+
+const config = require('../conf/Config');
+const zkConfig = config.zookeeper;
+const kafkaConfig = config.kafka;
+const extConfigs = config.extensions;
+const qpConfig = config.queuePopulator;
+const httpsConfig = config.internalHttps;
+const mConfig = config.metrics;
+const rConfig = config.redis;
```
Comment (tmacro) [inline bin/notification.js:13]: Alternate syntax, but I'm fine with either.
```suggestion
const {
    zookeeper: zkConfig,
    kafka: kafkaConfig,
    extensions: extConfigs,
    queuePopulator: qpConfig,
    internalHttps: httpsConfig,
    metrics: mConfig,
    redis: rConfig,
} = config;
```

**Review by tmacro — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e reset

**Comment by naren-rajendran** [comment]
@bert-e approve

**Comment by naren-rajendran** [comment]
@bert-e approve

**Pull request merged**